src/metabase/search/filter.clj

```diff
@@ -22,9 +22,12 @@
    :table-db-id         :database-id
    :verified            :verified})
 
+;; TODO dry this alias up with the index hydration code
+(def ^:private field-alias {:created-at :model-created-at})
+
 (def ^:private attr->index-key
   (into {} (for [k (vals context-key->attr)]
-             [k (keyword (str "search_index." (u/->snake_case_en (name k))))])))
+             [k (keyword (str "search_index." (u/->snake_case_en (name (field-alias k k)))))])))
```

Comment:
We have never had this discussion before, but I low-key hate when non-literal data structures are used as a function. When it's a literal (a keyword, a map - I don't care) — it's very obvious. When it's a name... IDK, somehow rubs me wrong, you don't get the understanding that it's not a function. I'd prefer `(get field-alias k k)` here for clarity if you don't mind. :)

Reply:
Sometime I think it's nice to think of the variable representing a function, which just happens to be represented by a map. Typically we call the something like `a->b` and don't pass a default arg though.

In this case I totally agree with you.